### PR TITLE
implement travel variables with travel tips

### PR DESCRIPTION
**Pull request by dulinajayarathna**

### CODE REVIEW

File: info/views.py
Comment (HarshaAbeyvickrama) [inline info/views.py:1]: perfect , merging into master

File: info/views.py [hunk 1]
```diff
@@ -4,7 +4,11 @@
 
 # Create your views here.
```
Comment (HarshaAbeyvickrama) [inline info/views.py]: remove this unwanted comment

**Review by HarshaAbeyvickrama — CHANGES_REQUESTED** [review]

File: info/views.py (hunk 1 above)
Comment (dulinajayarathna) [reply info/views.py]: Done

File: info/views.py [hunk 2]
```diff
@@ -2,9 +2,12 @@
 from django.shortcuts import render
```
Comment (HarshaAbeyvickrama) [inline info/views.py]: remove this unused import

**Review by HarshaAbeyvickrama — CHANGES_REQUESTED** [review]

File: info/views.py (hunk 2 above)
Comment (dulinajayarathna) [reply info/views.py]: done

**Comment by HarshaAbeyvickrama** [comment]
merging into master

**Pull request merged**